velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -485,6 +491,85 @@ TEST_F(DateTimeFunctionsTest, yearTimestampWithTimezone) {
           "year(c0)", std::nullopt, std::nullopt));
 }
 
+TEST_F(DateTimeFunctionsTest, weekDate) {
+  const auto weekDate = [&](const StringView& date) {
+    auto week =
+        evaluateOnce<int64_t>("week(c0)", std::make_optional(parseDate(date)))
```

Comment:
> std::make_optional(parseDate(date))

This code is rather long and repeats. Consider extracting a temp variable. Ditto other tests.

Reply:
Done, thanks!